include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +330,24 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     * Same as above, but using a of vector local degree-of-freedom values.
```

Comment:
FFS How did I manage that (esp. after I "double checked" this line before copy-pasting it elsewhere)?